client/app/src/pages/analyst/reports/reports.component.ts

```diff
@@ -0,0 +1,564 @@
+import { Component, OnInit } from "@angular/core";
```

Comment:
![Screenshot from 2024-11-15 19-15-30](https://github.com/user-attachments/assets/588fdd62-5dbd-4296-a9db-e2afc4b37508)

Notes:
- missing details on how reports are defined (format)
- missing details on how reports format could be saved
- missing details on how a report including its diagrams  could be exported / printed